test/TestAppUtils/VisualTreeDumper.cs

```diff
@@ -142,7 +142,7 @@ private static void WalkThroughTree(DependencyObject node, Visitor visitor)
         public class DefaultFilter : IFilter
         {
             private static readonly string[] _propertyNamePostfixWhiteList = new string[] {"Brush", "Thickness"};
-            private static readonly string[] _propertyNameWhiteList = new string[] {"Background", "Foreground", "Padding", "Margin", "RenderSize", "Visibility", "Name"};
+            private static readonly string[] _propertyNameWhiteList = new string[] {"Background", "Foreground", "Padding", "Margin", "RenderSize", "Visibility", "Name", "CornerRadius", "Width", "Height" };
```

Comment:
>"Width", "Height" [](start = 185, length = 17)

If we're going to include these then maybe we should also have Min|Max/Width|Height ? But since we have the RenderSize then I'm not sure we actually need it. 

Or if we do include it then can we not emit them when they're "NaN"?

Reply:
Added those in, I guess it doesn't hurt to have some redundancies. min==0, max==∞, NaN and empty strings are ignored now.